content/kotlin/concepts/datetime/terms/toDatePeriod/toDatePeriod.md

```diff
@@ -0,0 +1,42 @@
+---
+Title: 'toDatePeriod()'
+Description: 'Parses the ISO-8601 duration representation as a DatePeriod.'
+Subjects:
+  - 'Date and Time'
+  - 'Computer Science'
```

Comment:
```suggestion
  - 'Computer Science'
  - 'Code Foundations'
```
Again same issue check valid [subjects](https://github.com/Codecademy/docs/blob/main/documentation/subjects.md) before adding them in metadata.